frontend/packages/console-shared/src/components/utils/with-resources.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from 'react';
```

Comment:
There doesn't seem to be a corresponding code removal in context of the `public` directory, where does this code originate from?

@rawagner Is this code used in `kubevirt/web-ui` fork?

Reply:
Right, the code is coming from kubevirt/web-ui, I had to add it as hosts listing depends on it. I am happy to split it into a separate PR and update this one to depend on it if needed.